### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jialez0**
@sameo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,56 @@ entry point, also known as a [Relying Party](https://www.ietf.org/archive/id/dra
 in [RATS](https://datatracker.ietf.org/doc/draft-ietf-rats-architecture/)
 role terminology.
 
+KBS integrates the [Attestation-Service](https://github.com/confidential-containers/attestation-service) to verify TEE evidence.
+
+KBS can also be deployed as [Verifier](https://www.ietf.org/archive/id/draft-ietf-rats-architecture-22.html) in RATS role terminology.
```
Comment (sameo) [inline README.md:10]:
```suggestion
KBS can also be deployed as [RATS Verifier](https://www.ietf.org/archive/id/draft-ietf-rats-architecture-22.html).
```

File: README.md [hunk 2]
```diff
@@ -5,7 +5,56 @@ entry point, also known as a [Relying Party](https://www.ietf.org/archive/id/dra
...
+In this case, KBS will be responsible for distributing the Attestation Token (Passport).
```
Comment (sameo) [inline README.md:11]:
```suggestion
In this case, KBS will be responsible for distributing the Attestation Token (Following the RATS Passport model).
```

File: README.md [hunk 3]
```diff
@@ -5,7 +5,56 @@ entry point, also known as a [Relying Party](https://www.ietf.org/archive/id/dra
...
+
 ## Protocol
 
 The KBS implements and supports a simple, vendor and hardware-agnostic
 [implementation protocol](https://github.com/confidential-containers/kbs/blob/main/docs/kbs_attestation_protocol.md).
+
+## API
+
+KBS API is implemented based on HTTP/HTTPS. OpenAPI 3.1 description document:
```
Comment (sameo) [inline README.md:20]:
```suggestion
KBS implements an HTTP-based, [OpenAPI 3.1](https://spec.openapis.org/oas/v3.1.0) compliant API.
This API is formally described in its [OpenAPI formatted specification](docs/kbs.yaml).
```

File: README.md [hunk 4]
```diff
@@ -5,7 +5,56 @@ entry point, also known as a [Relying Party](https://www.ietf.org/archive/id/dra
...
+### Build and Run
+
+Start KBS and specify the address it listens to (take `127.0.0.1:8080` as an example):
+
+```shell
+make kbs
+./target/debug/kbs --socket 127.0.0.1:8080
+```
+
+Start with customized [config](src/config.rs):
```
Comment (sameo) [inline README.md:34]:
```suggestion
A custom, [JSON-formatted configuration file](src/config.rs) can be used:
```

File: README.md [hunk 5]
```diff
@@ -5,7 +5,56 @@ entry point, also known as a [Relying Party](https://www.ietf.org/archive/id/dra
...
+
+```shell
+./target/debug/kbs --socket 127.0.0.1:8080 --config /path/to/config.json
+```
+
+## Resource Repository
+
```
Comment (sameo) [inline README.md:41]: Here I would add one paragraph explaining what a resource repository is.

File: README.md [hunk 6]
```diff
@@ -5,7 +5,56 @@ entry point, also known as a [Relying Party](https://www.ietf.org/archive/id/dra
...
+KBS supports a variety of repository implementations.
+Which one to use is specified by config at startup (the default is the local file system)
+
+### Local File System Repository
+
+The mapping relationship between resource URL and resource file is:
```
Comment (sameo) [inline README.md:47]:
```suggestion
Resource files path map to a KBS resource URLs, as follows:

| Resource File Path  | Resource URL |
| ------------------- | -------------- |
| file://<$(KBS_REPOSITORY_DIR)>/<repository_name>/<type>/<tag>  |  http://<kbs_address>/kbs/v0/resource/<repository_name>/<type>/<tag>  |

```

File: README.md (hunk 5 above)
Comment (jialez0) [reply README.md:41]: Good suggestion, I will add this.

**Review by sameo — APPROVED** [review]

**Pull request merged**